packages/app-desktop/gui/styles/dialog-modal-layer.scss

```diff
@@ -23,6 +24,6 @@
 	}
 
 	&::backdrop {
-		background-color: rgba(0,0,0,0.6);
+		background-color: rgba(0,0,0,0.5);
```

Comment:
This CSS change prevents this pull request from changing the backdrop color for plugin dialogs.